docs/Lifecycle/DefinitionOfDone.md

```diff
@@ -0,0 +1,115 @@
+# Definition of Done (DoD)
+According SDL Contribution result of implementation should b a Pull Request to one or many SDL repositories.<br>
+Every Pull Requests should have following description:<br>
+> **Pull requests** *\<other pull requests which contain additional deliverables, if applicable\>*<br>
+> - Pull request 1<br>
+> - Pull request 2<br>
+> ...<br>
+> **Tasks** *\<implemented technical tasks/defects\>*<br>
+> - Technical task 1<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> ...<br>
+> - Technical task 2<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> ...<br>
+> - Defect 1<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> - Defect 2<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> ...<br>
+
+Following is the checklist of activities that add verifiable/demonstrable value to the product.<br>
+Each and every delivery to SDL should be verified for readiness upon this checklist.<br>
+DoD is not static; it evolves with SDL.<br>
+It is required to use this document as a checklist each time preparing delivery.<br>
+
+## Delivery completeness checklist
+Complete delivery to SDL consist of following deliverables:
+1. Source codes
+2. Code comments
```

Comment:
I think we can squash 1 and 2 points with link to [Pull request checklist](https://github.com/smartdevicelink/sdl_core/blob/develop/COMMITTERS.md#pull-request-checklist)